exotica/resources/configs/example_manipulate_ompl.xml

```diff
@@ -10,7 +10,6 @@
 
     <PlanningScene>
       <Scene>
-        <!-- Computing distances is enabled by default, disable for a Sampling problem by setting the PlanningMode to speed up Scene updates -->
```

Comment:
I think this comment was helpful for people using the example configs to learn about how the system works.

Reply:
That is a deprecated parameter. I removed it from the XML.